### ISSUE TITLE
Pod install issue

### ISSUE THREAD

**Issue by mcagrigoktas** [tanersener/flutter-ffmpeg#8]
Error output from CocoaPods:
↳
 [!] Automatically assigning platform `ios` with version `9.3` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error running pod install
Error launching application on iPhone XS.

**Comment by tanersener** [comment]
Do you have a platform definition like `platform :ios, '9.3'` in your `Podfile`?

**Comment by PiotrWpl** [comment]
I believe, the main issue is not a platform version, but some static binaries.
I tried to use this plugin in project I'm working on, and it's working great in Android, but unfortunately it's not compiling on iOS. This is what I got as output:

```
Running pre install hooks
    [!] The 'Pods-Runner' target has transitive dependencies that include static binaries: (/Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/mobileffmpeg.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libavcodec.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libavdevice.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libavfilter.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libavformat.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libavutil.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libswresample.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libswscale.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/gmp.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/gnutls.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libcharset.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libhogweed.framework, /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libiconv.framework, and /Users/piotrw/Development/myProject-flutter/ios/Pods/mobile-ffmpeg-https/libnettle.framework)

    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer/xcode/target_validator.rb:81:in `block (2 levels) in verify_no_static_framework_transitive_dependencies'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer/xcode/target_validator.rb:73:in `each'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer/xcode/target_validator.rb:73:in `block in verify_no_static_framework_transitive_dependencies'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer/xcode/target_validator.rb:70:in `each'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer/xcode/target_validator.rb:70:in `verify_no_static_framework_transitive_dependencies'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer/xcode/target_validator.rb:36:in `validate!'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer.rb:415:in `validate_targets'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer.rb:118:in `install!'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/command/install.rb:41:in `run'
    /Library/Ruby/Gems/2.3.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/command.rb:52:in `run'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/bin/pod:55:in `<top (required)>'
    /usr/local/bin/pod:22:in `load'
    /usr/local/bin/pod:22:in `<main>'

Error output from CocoaPods:
↳

    [!] Automatically assigning platform `ios` with version `10.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error running pod install
Error launching application on iPhone XR.
```

In my opinion the part above the info about platform is more important, and the main reason of build error. Of course I tried to add:
> platform :ios, '10.0'
to my Podfile, and it fixed bottom error (not included in this example log), but the one about static binaries left :/

I also tried solution from this issue https://github.com/flutter/flutter/issues/20045
to comment:
> use_frameworks!
but this didn't helped :/ I got a lot of other type of errors from other plugins.

I'm not to advanced iOS developer, maybe there is some way to recompile iOS part, and then this plugin will work?

BTW: Greate plugin! It really helped me in my project, if only I would have iOS working :(

**Comment by tanersener** [comment]
> I'm not to advanced iOS developer, maybe there is some way to recompile iOS part, and then this plugin will work?
> 
> BTW: Greate plugin! It really helped me in my project, if only I would have iOS working :(

There is a working test application under `example` folder which works for `IOS` too. Can your share your full `pubspec.yaml` file so I can see the difference?

Additionally, `development` branch includes a few changes, do you receive the same error when you use the following dependency.

```
  flutter_ffmpeg:
    git:
      url: git://github.com/tanersener/flutter-ffmpeg.git
      ref: development
```

**Comment by PiotrWpl** [comment]
Hi!
I added version from development branch and did a few more tests. And looks like it's working now!

Changing flutter_ffmpeg wasn't enough, I got a couple of different errors than in stable release.

After that I checked the difference between ours Podfiles, and of course it was lack of "use_frameworks!" in yours file from example app.

So I decided to remove it once again from my version, and disable Flutter plugins (simple_share, flutter_inappbrowser) which stopped working after this change.
After that project compiled successfully, and yours plugin works as expected in my iOS simulator.

Thank You for your fast response and help!
I really appreciate that! Cheers!
